src/ci/docker/host-x86_64/x86_64-gnu-integration/build-fuchsia.sh

```diff
@@ -2,50 +2,73 @@
 
 # Downloads and builds the Fuchsia operating system using a toolchain installed
 # in $RUST_INSTALL_DIR.
+#
+# You may run this script locally using Docker with the following command:
+#
+# $ src/ci/docker/run.sh x86_64-gnu-integration
 
 set -euf -o pipefail
 
-INTEGRATION_SHA=1011e3298775ee7cdf6f6dc73e808d6a86e33bd6
+# Set this variable to disable updating the Fuchsia checkout. This is useful for
+# making local changes. You can find the Fuchsia checkout in `obj/fuchsia` in
+# your local checkout after running this job for the first time.
+KEEP_CHECKOUT=
+
+# Any upstream refs that should be cherry-picked. This can be used to include
+# Gerrit changes from https://fxrev.dev during development (click the "Download"
+# button on a changelist to see the cherry pick ref). Example:
+# PICK_REFS=(refs/changes/71/1054071/2 refs/changes/74/1054574/2)
 PICK_REFS=()
 
+# The commit hash of integration.git to check out (this controls the commit hash
```

Comment:
Should this get bumped regularly or what

Reply:
Yes that is already the case, [every few months or so](https://github.com/rust-lang/rust/commits/master/src/ci/docker/host-x86_64/x86_64-gnu-integration/build-fuchsia.sh) but is probably a good callout.

Reply:
Yeah, we aim for once every 1-2 months right now